### PR TITLE
Rails Capstone Project: FINAL REVIEW

### PR DESCRIPTION
**Pull request by AbdimulhinYussuf3675**
### WELCOME TO BUDGET APP
**During this capstone project, I successfully achieved the following milestones:**

- I used `Ruby and Ruby on Rails (RoR)` design this RESTful application.
- The application includes several models based on an [Entity-Relationship Diagram (ERD) in PostgresSQL](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png). The models include
- User, Group, Expense, and a bridge tables.
- Authentication is implemented using Devise, while authorization is handled by cancancan.
- An unauthorized user is presented with a splash screen upon accessing the application.
- The category page serves as the home page for the application.
- A transaction page has been created.
- Unit and integration tests have been written using RSpec.
- Authenticated users have the ability to create groups and expenses.
- I have recorded a video presentation showcasing the project's details. please check below
- The web service has been deployed on the Render site, as shown below
- The design of the application adhered to a mobile-first approach
### Project presentation video record_button . Please click the link to check it out [`link`](https://drive.google.com/file/d/1aRpppoBC4nVBg6iR4VZo6LaX34jCnjJj/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,165 @@
...
+- **User can create an account..**
+- **Registered users will be able to create new category and operation..**
+- **Users will be able to view operations and category on the application's home page.**
+- **User can login..**
+- **Rspec test for models, controllers and views..**
+
+## 🚀 Project documentation 
+
+
+### Take a look at project's [Live Demo](https://budgetapp-7px9.onrender.com/)
```
Comment (BregornOriginal) [inline README.md]: - [ ] Kindly fix the link to your live version, there's a black screen on that link

File: README.md [hunk 2]
```diff
@@ -1,24 +1,165 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like the project, give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
```
Comment (BregornOriginal) [inline README.md:157]: - [ ] As was requested in the requirements, kindly gives in this section [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @AbdimulhinYussuf3675  

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

giphy (5)

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by addod19 — APPROVED** [review]
### Hi @AbdimulhinYussuf3675, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the deployed link :+1: 

**NB**: YOU DID NOT FOLLOW THE CORRECT GIT-FLOW WORKFLOW BUT INSTEAD OF THE PROJECT BEING INVALIDATED, IT HAS BEEN REVIEWED. PLEASE TRY AND FOLLOW PROJECT GUIDELINES. GIT-FLOW MEANS CREATING A PR FROM A DESCRIPTIVE FEATURE BRANCH, FOR EXAMPLE `BUDGET` OR ANY DESCRIPTIVE NAME YOU MAY CHOOSE INTO THE `DEV` BRANCH. THAT IS GIT-FLOW


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdimulhinYussuf3675** [comment]
Thanks @addod19,  Sure I will fix it

**Pull request merged**